content/cucumber/step-definitions.md

```diff
@@ -253,3 +253,48 @@ Given("I have {int} {color} balls", function (int1, color) {
 });
 ```
 {{% /block %}}
+
+# Abstract Your Steps
```

Comment:
I'd prefer to not use the term "abstract" (to avoid confusion with the java keyword).
Maybe it should be called "helper methods" - which is fact might be a better location: https://docs.cucumber.io/gherkin/step-organization/#helper-methods
And as it is asked more often, add a question "How do I group my step definitions?" with a brief answer and link to this info.

Reply:
Didn't understand you completely. Do you want me to move this to the **helper-methods** section and link to there from the **step-definitions** page? Or the other way round?

Reply:
Merge with the helper-method section please. :)

Reply:
Oh, turns out we already have a section on Helper methods in the FAQ too: https://docs.cucumber.io/community/faq/#how-to-use-helper-methods

So what I meant before was, there should be a question in the FAQ page (there is), which should give a brief answer to the question (it does) and point to the relevant section in the docs (https://docs.cucumber.io/gherkin/step-organization/#helper-methods). Like we do for all the other FAQ.

Reply:
Alright. I have made some updates here.
Once we finalize the content, I ll move it to the new section and link it from the FAQs.

Reply:
Content is fine; I suggest merging with the other section; we might need some edits if there's any duplication. :)

Reply:
Ofcourse. Let me merge this with that section then.

Reply:
Ok so i updated the FAQ and actually removed the example from there to keep it brief. We already provide a good example in the `group step definitions` section, so I removed it from the FAQ.

If you think we should keep it, I would suggest putting it inside the `Helper methods` section of the Step Organization tab instead. To keep all the info regarding helper methods at one place.